compareECLFiles.cmake

```diff
@@ -202,14 +202,19 @@ add_test_compareECLFiles(CASENAME ctaquifer_2d_oilwater
                          REL_TOL ${rel_tol}
                          DIR aquifer-oilwater)
 
-foreach(SIM flow flow_legacy)
-  add_test_compareECLFiles(CASENAME spe3
-                           FILENAME SPE3CASE1
-                           SIMULATOR ${SIM}
-                           ABS_TOL ${abs_tol}
-                           REL_TOL ${coarse_rel_tol}
-                           TEST_ARGS tolerance_wells=1e-6 max_iter=20)
-endforeach()
+add_test_compareECLFiles(CASENAME spe3
+                         FILENAME SPE3CASE1
+                         SIMULATOR flow
+                         ABS_TOL ${abs_tol}
+                         REL_TOL ${coarse_rel_tol}
+                         TEST_ARGS --flow-tolerance-wells=1e-6 --flow-newton-max-iterations=20)
```

Comment:
maybe `flow-newton-max-iterations=20` is the thing change the iteration behavoir?